packages/fast-usdc/src/exos/liquidity-pool.js

```diff
@@ -88,6 +88,10 @@ export const prepareLiquidityPoolKit = (zone, zcf, USDC, tools) => {
           SeatShape,
           harden({ USDC: makeNatAmountShape(USDC, 1n) }),
         ).returns(),
+        repay: M.call(
```

Comment:
using the same name (`repay`) with different semantics (sometimes fees, sometimes without) is asking for trouble, IME.

maybe...

```suggestion
        returnToPool: M.call(
```

```suggestion
        repayFailed: M.call(
```

Reply:
Noted, updated to `returnToPool`